.taprc

```diff
@@ -1,6 +1,4 @@
-100: true
```

Comment:
Just one question:

Why did we go to `disable-coverage` from `100`?

Maybe it works without `disable-coverage`?